recipe/windows_cmake_install_bindir.patch

```diff
@@ -0,0 +1,37 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 3fd9d70..f8bf221 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -399,6 +399,7 @@ endif ()
+ if (OS_WINDOWS)
+   set (RUNTIME_INSTALL_DIR Bin)
+   set (LIBRARY_INSTALL_DIR Lib)
```

Comment:
Yes, now it is clear. `.dll`s get placed in `RUNTIME_INSTALL_DIR` (`bin` in our case). Thanks for clearing that up for me.